app/js/components/bullhorn/bullhorn.jade

```diff
@@ -4,15 +4,13 @@ span.dropdown.dropdown-notification(dropdown='', on-toggle='toggled(open)')
     span.count(ng-if="unackedCount !== 0") {{unackedCount}}
   ul.dropdown-menu.preview
     li.announcement(ng-repeat="item in items", ng-class="{ acked: (item.acked) }", ng-click="item.acked ? selectAnnouncement(item) : ack(item)")
-      .row
-        .col-sm-12
+      .announcement-message
+        .title
           b {{item.title}}
-          .message-status-toggle
-            i.fa.fa-check-circle-o.acked(ng-if="item.acked")
-            i.fa.fa-circle-o(ng-if="!item.acked")
-      .row
-        .col-sm-12
-          .message {{item.body.length > 140 ? item.body.substring(0, 140) + '...' : item.body.substring(0, item.body.length)}}
+        .message {{item.body.length > 140 ? item.body.substring(0, 140) + '...' : item.body.substring(0, item.body.length)}}
```

Comment:
There is a `trimmed-text` component that should be a good fit to be reused here https://github.com/RedHatInsights/insights-frontend/blob/master/app/js/components/misc/trimmedText/trimmedText.directive.js